packages/wrangler/src/pages/dev.ts

```diff
@@ -75,6 +75,19 @@ const SERVICE_BINDING_REGEXP = new RegExp(
 	/^(?<binding>[^=]+)=(?<service>[^@#\s]+)(@(?<environment>.*)$)?(#(?<entrypoint>.*))?$/
 );
 
+/* DISPATCH_NAMESPACE_REGEXP matches strings like:
+ * - "BINDING=ref"
+ * - "BINDING=ref@outbound"
+ * - "BINDING=ref@outbound#parameter,parameter2"
+ * - "BINDING=ref@outbound@envrionment"
+ * - "BINDING=ref@outbound@envrionment#parameter,parameter2"
+ * This is used to capture both the binding name (how the binding is used in JS) as well as the dispatch namespace reference.
+ * Optionally, it can include an outbound service name (with optional environment) and optionally parameters as well.
+ */
+const DISPATCH_NAMESPACE_REGEXP = new RegExp(
+	/^(?<binding>[^=]+)(?:=(?<ref>[^\s@#]+))(@(?<outbound>[^\s@#]+)(@(?<environment>[^\s#]+))?(#(?<parameters>[^\s]+)))?$/
```

Comment:
oh boy... can we just use this as an opportunity to push people towards wrangler.toml? Not support through cli and only wrangler.toml